packages/rollup-plugin-index-html/test/integration.test.js

```diff
@@ -33,6 +33,12 @@ async function testSnapshot(name, configFilePath) {
     });
   }
 
+  if (name.includes('inline-entry')) {
```

Comment:
`testSnapshots` below already does the same I tink

Reply:
this whole block was committed by mistake